dsl/src/commonMain/kotlin/it/unibo/collektive/field/Field.kt

```diff
@@ -104,27 +104,78 @@ sealed interface Field<ID : Any, out T> {
 
         /**
          * Reduce the elements of the field using the [transform] function.
-         * The local value is not considered, unless explicitly passed as [default].
+         * The local value is not considered.
```

Comment:
```suggestion
         * Reduce the elements of the field using the [transform] function,
         * returning the [default] value if the field to transform is empty.
         * The local value is not considered.
```

Then remote the last line